tracer/time_windows.go

```diff
@@ -0,0 +1,31 @@
+package tracer
+
+import (
+	"golang.org/x/sys/windows"
+	"time"
+)
+
+// This method is more precise than the go1.8 time.Now on Windows
+// See https://msdn.microsoft.com/en-us/library/windows/desktop/hh706895(v=vs.85).aspx
+// It is however ~10x slower and requires Windows 8+.
```

Comment:
Yes this imprecision is currently an issue.
@bmermet did you check that `GetSystemTimePreciseAsFileTime` raises much more precise results? This initial issue was that `time.Now()` could easily return `0` — maybe that's something we could easily add a test for?